builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -158,6 +173,18 @@ func (b *backend) pathUpdateIssuer(ctx context.Context, req *logical.Request, da
 	}
 
 	newPath := data.Get("manual_chain").([]string)
+	rawLeafBehavior := data.Get("leaf_not_after_behavior").(string)
```

Comment:
Does this not fail the update api call if we don't specify the `leaf_not_after_behavior` argument?

Reply:
Yup you are correct I keep forgetting about the defaults...